### PR TITLE
Restructure packaging for await component

### PR DESCRIPTION
**Pull request by etman55**
image

COAND-497

### CODE REVIEW

File: await/src/main/java/com/adyen/checkout/await/internal/ui/model/AwaitOutputData.kt [hunk 1]
```diff
@@ -5,10 +5,12 @@
  *
  * Created by caiof on 25/8/2020.
  */
-package com.adyen.checkout.await
+package com.adyen.checkout.await.internal.ui.model
 
+import androidx.annotation.RestrictTo
 import com.adyen.checkout.components.base.OutputData
 
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```
Comment (jreij) [inline await/src/main/java/com/adyen/checkout/await/internal/ui/model/AwaitOutputData.kt:13]: Can't this be internal?

File: await/src/main/java/com/adyen/checkout/await/internal/ui/AwaitDelegate.kt [hunk 2]
```diff
@@ -6,14 +6,17 @@
...
 
+import androidx.annotation.RestrictTo
+import com.adyen.checkout.await.internal.ui.model.AwaitOutputData
 import com.adyen.checkout.components.base.ActionDelegate
 import com.adyen.checkout.components.base.DetailsEmittingDelegate
 import com.adyen.checkout.components.base.StatusPollingDelegate
 import com.adyen.checkout.components.base.ViewableDelegate
 import com.adyen.checkout.components.ui.ViewProvidingDelegate
 
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```
Comment (jreij) [inline await/src/main/java/com/adyen/checkout/await/internal/ui/AwaitDelegate.kt:19]: Can this be internal?

Comment (etman55) [reply await/src/main/java/com/adyen/checkout/await/internal/ui/AwaitDelegate.kt:19]: No as we pass it through the component constructor.

File: await/src/main/java/com/adyen/checkout/await/internal/ui/model/AwaitOutputData.kt (hunk 1 above)
Comment (etman55) [reply await/src/main/java/com/adyen/checkout/await/internal/ui/model/AwaitOutputData.kt:13]: This is used inside `AwaitDelegate` and I need to make `AwaitDelegate` internal to make it work and I can't make the delegate internal.

**Review by jreij — APPROVED** [review]

**Pull request merged**